Readme.md

```diff
@@ -112,12 +112,16 @@ echo $waiter->isSuccess(); // true
 
 | Package name                                                                  | Badges                                                                                                                                                                                                                                                                                                  | BC check                  |
 | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------- |
+| [async-aws/code-generator](https://github.com/async-aws/code-generator)       | [![Latest Stable Version](https://poser.pugx.org/async-aws/code-generator/v/stable)](https://packagist.org/packages/async-aws/code-generator)        [![Total Downloads](https://poser.pugx.org/async-aws/code-generator/downloads)](https://packagist.org/packages/async-aws/code-generator)            | [![](https://github.com/async-aws/code-generator/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/code-generator/actions)
```

Comment:
I'm not sure this package should be exposed on main page. People are looking for AWS services don't you think?